src/Microsoft.Health.Fhir.Core/Features/Operations/IArtifactProvider.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations
+{
+    public interface IArtifactProvider
+    {
+        Task FetchAsync(string location, Stream targetStream, CancellationToken cancellationToken);
```

Comment:
Perhaps add documentation to this interface for future reference

Reply:
Thanks, updated.